<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>866</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GraficThink</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>globe_earth_planet_2767.png</normaloff>globe_earth_planet_2767.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>841</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>281</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>33</y>
        <width>191</width>
        <height>344</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="janeiro">
         <property name="text">
          <string>Janeiro</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="dezembro">
         <property name="text">
          <string>Dezembro</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="fevereiro">
         <property name="text">
          <string>Fevereiro</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="novembro">
         <property name="text">
          <string>Novemnro</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="marco">
         <property name="text">
          <string>Março</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="abril">
         <property name="text">
          <string>Abril</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="junho">
         <property name="text">
          <string>Junho</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="maio">
         <property name="text">
          <string>Maio</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="outubro">
         <property name="text">
          <string>Outubro</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="julho">
         <property name="text">
          <string>Julho </string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="agosto">
         <property name="text">
          <string>Agosto</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="setembro">
         <property name="text">
          <string>Setembro</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fatJaneiro">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fatFevereiro"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="fatMarco"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="fatAbril"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="fatMaio"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="fatJunho"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="fatJulho"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="fatAgosto"/>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="fatSetembro"/>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="fatOutubro"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="fatNovembro"/>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="fatDezembro"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="gerargraficos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gerar Grafico</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="gerargraficos_2">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>400</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar Campos</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Valores</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Meses</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>19</y>
       <width>481</width>
       <height>431</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Avisos</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>361</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Anotações</string>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>361</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="gerargraficos_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>222</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>SALVAR DB</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>janeiro</tabstop>
  <tabstop>fatJaneiro</tabstop>
  <tabstop>fevereiro</tabstop>
  <tabstop>fatFevereiro</tabstop>
  <tabstop>marco</tabstop>
  <tabstop>fatMarco</tabstop>
  <tabstop>abril</tabstop>
  <tabstop>fatAbril</tabstop>
  <tabstop>maio</tabstop>
  <tabstop>fatMaio</tabstop>
  <tabstop>junho</tabstop>
  <tabstop>fatJunho</tabstop>
  <tabstop>julho</tabstop>
  <tabstop>fatJulho</tabstop>
  <tabstop>agosto</tabstop>
  <tabstop>fatAgosto</tabstop>
  <tabstop>setembro</tabstop>
  <tabstop>fatSetembro</tabstop>
  <tabstop>outubro</tabstop>
  <tabstop>fatOutubro</tabstop>
  <tabstop>novembro</tabstop>
  <tabstop>fatNovembro</tabstop>
  <tabstop>dezembro</tabstop>
  <tabstop>fatDezembro</tabstop>
  <tabstop>gerargraficos</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatJaneiro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatFevereiro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatMarco</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatAbril</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatMaio</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatJunho</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatJulho</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatAgosto</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatSetembro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatOutubro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatNovembro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerargraficos_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>fatDezembro</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
